sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - The `IDisposable` interface has been removed from `TokenCache`.
 
+### New Features
+
+- All credentials now support CAE claims challenges
```

Comment:
I don't think we should call this out a specific to CAE in the release notes. I think it should more generally state that all credentials added support to handle the `Claims` property on `TokenReqeustContext`